core/src/main/resources/defaults/clustertemplates/7.2.10/aws/edgenode.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "7.2.10 COD Edge Node for AWS",
+  "description": "",
+  "type": "OPERATIONALDATABASE",
+  "featureState": "RELEASED",
+  "cloudPlatform": "AWS",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "7.2.10 COD Edge Node"
+    },
+    "instanceGroups": [
+      {
+        "name": "leader",
+        "nodeCount": 1,
+        "recoveryMode": "MANUAL",
+        "template": {
+          "attachedVolumes": [
+            {
+              "count": 1,
+              "size": 100,
+              "type": "gp2"
+            }
+          ],
+          "aws": {
+            "encryption": {
+              "type": "NONE"
+            }
+          },
+          "instanceType": "m5.4xlarge"
```

Comment:
The m5.4xlarge has 16 vCPUs and 64 GiB memory but Standard_D8_v3 has half of these (8 vCPU, 32Gib).
Can we rather go with m5.2xlarge for AWS (8 vCPU, 32GiB) to be in sync? I don't think m5.4xlarge is really required and on Edge node creation the customer could still increase the instance size.

Reply:
Sure, good point.